src/main/java/me/zeroeightsix/kami/gui/hudgui/LabelHud.kt

```diff
@@ -5,16 +5,15 @@ import me.zeroeightsix.kami.util.graphics.VertexHelper
 import me.zeroeightsix.kami.util.graphics.font.TextComponent
 import me.zeroeightsix.kami.util.math.Vec2d
 import me.zeroeightsix.kami.util.threads.safeAsyncListener
+import me.zeroeightsix.kami.util.translation.TranslationKey
 import net.minecraftforge.fml.common.gameevent.TickEvent
 
 abstract class LabelHud(
-    name: String,
     alias: Array<String> = emptyArray(),
     category: Category,
-    description: String,
     alwaysListening: Boolean = false,
     enabledByDefault: Boolean = false
-) : HudElement(name, alias, category, description, alwaysListening, enabledByDefault) {
+) : HudElement( alias, category, alwaysListening, enabledByDefault) {
```

Comment:
Not sure why you're removing these

Reply:
I have removed those because they will be generated automatically from a file later, and having them here will just lead them to be overwritten.